app/templates/events/view/speakers.hbs

```diff
@@ -16,19 +16,12 @@
         {{/link-to}}
       {{/tabbed-navigation}}
     </div>
-    <div class="eight wide column">
-      <button class="ui blue button right floated">{{t 'Add Speaker'}}</button>
-      <button class="ui blue button right floated">{{t 'Export as CSV'}}</button>
-      {{#ui-dropdown class='labeled icon button right floated'}}
-        <i class="dropdown icon"></i>
-        <div class="default text">{{t 'No. of entries'}}</div>
-        <div class="menu">
-          <div class="item">10</div>
-          <div class="item">25</div>
-          <div class="item">50</div>
-          <div class="item">100</div>
-        </div>
-      {{/ui-dropdown}}
+    <div class="ten wide {{if device.isMobile 'center aligned'}} column">
+      <button class="ui blue button {{if device.isMobile 'fluid' 'right floated'}}">{{t 'Export as CSV'}}</button>
+      {{#if device.isMobile}}
+        <div class="ui fitted divider"></div>
```

Comment:
@harshitagupta30 Add `hidden` class to this, instead of simply removing `fitted`. Everything else looks good to me.